modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
+* *Applications* or *Servers*
++
+The application or server to apply the alert to
+* *Condition*
++
+The condition that triggers the alert
++
+The available options depend on whether your alert is linked to a server,
+an application running on CloudHub, or an application running locally.
+Some conditions require additional data.
+See <<Alert Conditions>>.
+* *Subject*
++
+The subject of the alert email to send when the alert is triggered.
++
+The subject can include variables.
+See <<Alert Properties>>.
+* *Message*
++
+The body text for the email to send when the alert is triggered.
+The message body can include variables.
+See <<Alert Properties>>.
+* *Recipients*
++
+The recipients of the alert email
++
+You can select platform users or enter any email address.
 
 . Click *Create Alert*.
 
-. If an event that meets an alert condition occurs, an email like the following is sent:
-+
-image::alertemailmsg.png[AlertEmailMsg]
+When an event that meets an alert condition occurs, an email like the following is sent:
 
+image::alerts-alert-email.png[Anypoint Platform Alert Notification email]
 
-See below for more information on the alert conditions available for each alert source.
 
 == Alert Conditions
 
-Depending on what your alert's source is, you can select different event types. Some event types are simple success or failure notifications such as 'Application Deployment Success' and more complex notifications which require you to define parameters that appear as new fields in the form.
+You can select different conditions depending on the alert source.
+Some conditions are simple success or failure notifications, such as `Deployment success`.
+Some more complex conditions require that you define parameters that appear as new fields in the form.
 
-=== Conditions on CloudHub Deployed Applications
+=== Conditions for Applications Deployed to CloudHub 
 
-For applications deployed to CloudHub, you can choose any of the following alert conditions:
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-* *CPU Usage* - CPU usage of the CloudHub worker(s) that run the application.
-+
-image::cpu-usage.png[cpu]
-+
-To enable this condition, you must specify the duration and capacity threshold that triggers the alert.
+For applications deployed to CloudHub, the following alert conditions apply:
 
-* *Memory Usage* - Percentage of total memory allocated to the CloudHub worker(s) that run the application (includes the memory consumed by the JVM itself).
+CPU usage - Cloudhub::
```

Comment:
Is this the actual name or displayed condition?
I ask only because we want to avoid using space-hyphen-space anywhere in our text.
GLOBAL in this list; I won't mark again.

Reply:
It's as it appears in the UI - see screenshot.